### PR TITLE
Order PipelineComponent fields

### PR DESCRIPTION
**Pull request by disrupted**

### CODE REVIEW

File: kpops/components/base_components/pipeline_component.py [hunk 1]
```diff
@@ -52,25 +50,21 @@ class PipelineComponent(BaseDefaultsComponent):
...
+        default="${pipeline_name}-",
+        description=describe_attr("prefix", __doc__),
+    )
     from_: FromSection | None = Field(
         default=None,
         alias="from",
         title="From",
-        description=describe_attr("from_", __doc__),
-    )
-    app: object | None = Field(
```
Comment (philipp94831) [inline kpops/components/base_components/pipeline_component.py:62]: Will this break things?

File: kpops/components/base_components/pipeline_component.py [hunk 2]
```diff
@@ -27,18 +27,16 @@ class PipelineComponent(BaseDefaultsComponent):
...
-    :param app: Application-specific settings, defaults to None
-    :type app: object, optional
-    :param to: Topic(s) into which the component will write output,
-        defaults to None
-    :type to: ToSection, optional
     :param prefix: Pipeline prefix that will prefix every component name.
         If you wish to not have any prefix you can specify an empty string.,
         defaults to "${pipeline_name}-"
     :type prefix: str, optional
+    :param from: Topic(s) and/or components from which the component will read
```
Comment (sujuka99) [inline kpops/components/base_components/pipeline_component.py:34]: Why did you rename `from_` to `from` in the docstr?

File: kpops/components/base_components/pipeline_component.py (hunk 1 above)
Comment (disrupted) [reply kpops/components/base_components/pipeline_component.py:62]: I'll test it. It seems like this attribute made it here by mistake or it stems from an old workaround

File: kpops/components/base_components/pipeline_component.py (hunk 2 above)
Comment (disrupted) [reply kpops/components/base_components/pipeline_component.py:34]: done by mistake, I thought the parameter can be used with the alias, but that's not the case. will revert

File: kpops/components/base_components/pipeline_component.py (hunk 1 above)
Comment (disrupted) [reply kpops/components/base_components/pipeline_component.py:62]: tested ✔️

**Review by raminqaf — APPROVED** [review]
